engine-precompiles/src/native.rs

```diff
@@ -237,17 +244,26 @@ impl<I: IO> Precompile for ExitToNear<I> {
         context: &Context,
         is_static: bool,
     ) -> EvmPrecompileResult {
+        // ETH transfer input format: (21+ bytes)
+        //  - flag (1 byte)
+        //  - refund_address (20 bytes)
+        //  - 1 byte ??
+        //  - recipient_account_id (N bytes)
+        // ERC20 transfer input format: (54+ bytes)
+        //  - flag (1 byte)
+        //  - refund_address (20 bytes)
+        //  - 1 byte ???
```

Comment:
I didn't get why in the case of `ExitToNear` there is an extra byte ?

cc @joshuajbouw, @birchmd Still @aleksuss